blog/2024-10-30-agama-10.mdx

```diff
@@ -0,0 +1,199 @@
+possible to install openSUSE Leap 16.0 Alpha. We also created a SLES 16 flavor but for now it is not
+available for public.
+
+The product selection dialog now contains the product icons so the product selection should be more
+intuitive.
+
+![Product selection](../static/img/blog/2024-10-30/product-selection.png)
+
+## Improvements in storage {#storage}
+
+Agama now supports creating LVM volume groups, logical volumes, thin pools and thin logical volumes
+in automatic installations. Reusing volume groups or logical volumes is not supported.
+
+Also the existing partitions can be deleted automatically. There are two options, unconditional
+delete and delete only if needed. The first option always deletes the specified partition, the
+second option deletes it only when there is not enough space to create the needed partitions.
+
+Always delete a partition:
+
+```json
+{
+  "search": "/dev/vda1",
+  "delete": true
+}
+```
+
+Delete a partition only when needed:
+
+```json
+{
+  "search": "/dev/vda1",
+  "deleteIfNeeded": true
+}
+```
+
+See more details in the [auto_storage](
+https://github.com/openSUSE/agama/blob/master/doc/auto_storage.md) documentation.
+
+## Installation repository URL {#installation-url}
+
+By default Agama installs the packages from the repositories specified in the product configuration.
+Those normally points to the official SUSE/openSUSE installation repositories.
+
+But for testing purposes, especially in the openQA tests, you might need to install the packages
+from some testing or staging repository to test the latest versions which are not published in those
+official repositories yet.
+
+From now Agama supports using the `agama.install_url` boot parameter for overriding the default
+installation repositories. You can use multiple URLs separated by comma:
+
+```bash
+agama.install_url=https://example.com/1,https://example.com/2
+```
+
+## Internal changes {#internal}
+
+There are lots of internal changes in Agama so we mention here only the most important ones.
+
+### Typescript support {#typescript}
+
+The Agama web UI started in plain Javascript. But that is quite fragile, you can easily by mistake
+pass wrong arguments to a function or call a missing method on an object. Such a mistake would cause
+a crash. The [Typescript](https://www.typescriptlang.org/) language avoids all these kinds of
+problems.
+
+Recently we added support for writing parts of the web UI in Typescript. We do not plan to do any
+big refactoring at once, the files will be gradually switched from Javascript to Typescript when
+touching them.
+
+### TanStack Query state management {#state-management}
+
+In the web frontend we started using the [TanStack Query](https://tanstack.com/query/latest)
+framework for the state management. It provides a mechanism based on promises to fetch or update the
+state automatically. It takes care of caching, re-fetching, error handling, etc. so you do not have
+to implement these features by yourselves.
```

Comment:
yourselves or yourself?

Reply:
Yourselves is plural which IMHO fits better in this case.